chainer/functions/math/fix.py

```diff
@@ -0,0 +1,40 @@
+from chainer import cuda
+from chainer import function_node
+from chainer import utils
+from chainer.utils import type_check
+
+
+class Fix(function_node.FunctionNode):
```

Comment:
Please remove this class.
Because `backward` of this class always returns `None`.